screwdriver.yaml

```diff
@@ -0,0 +1,6 @@
+jobs:
+    main:
+        steps:
+            echo: echo $SECRET_PASSWORD
```

Comment:
this is not a real password and we are just testing out the mounting of environment variables into the build